kqueen/blueprints/api/views.py

```diff
@@ -25,7 +25,7 @@
 import yaml
 
 config = current_config()
-logger = logging.getLogger(__name__)
+logger = logging.getLogger('kqueen_api')
```

Comment:
Why are you hidding and ignoring python logger hierarchy? We can use root logger and use propagation to catch all logs.

Reply:
__name__  var used only for module naming, i already defined *module* naming in logger formatting (logger_config.py), it will be good for:
1) reuse 1 logger and does not create multiple loggers -> increase performance
2) all logging arch stored in one file -> increase visibility

so in fact , no cons, just pros

Reply:
and its also needed in case of splitting user logging and common logging, but needed to provide common arch for api and ui parts , so please take a look on ui part

Reply:
Ok, thanks for clarification, your solution is easier and simple, let's use it.